### ISSUE TITLE
'working' state isn't being cleared when hot plate timer expires

### ISSUE THREAD

**Issue by brewston** [sergeymaysak/smartercoffee#3]
I *think* I've reproduced this a few times now Sergey , so think this is a bug.

When my machine finishes brewing, (if I get interrupted by a call before I get a chance to get to the kitchen), I will manually activate a hot plate timer for 5mins. Whilst the state changes to 'heating plate' for the 5mins, when the timer expires, the state changes to 'working' (When I would expect it to revert back to 'ready')

Thanks and I appreciate your development effort on this.

**Comment by sergeymaysak** [comment]
Fixed in [1.4.4](https://github.com/sergeymaysak/smartercoffee/releases/tag/1.4.4)

**Comment by brewston** [comment]
I don't think this fix worked I'm afraid. When I turn on the heating plate, the state transitions to 'heating plate' (Which is correct) but when the timer expires, I'd expect the state to transition back to 'ready' but instead it remains on 'heating plate'

**Comment by sergeymaysak** [comment]
@brewston reopened it. reproduced. sorry for the silence so long.  Not sure, however, when will have time to dig in into it. Thanks for testing it. ) I will definitely keep you posted.
